trello.py

```diff
@@ -0,0 +1,90 @@
+import os
+import pprint
+import requests
+
+pp = pprint.PrettyPrinter(indent=4)
+app_board_name='To Do App'
+trello_member='brendanmcdaniel'
+trello_endpoint='https://api.trello.com/1/'
+
+class item:
+    def __init__(self, id, status, title):
+        self.id = id
+        self.status = status
+        self.title = title
+
+class status:
+    def __init__(self, id, name):
+        self.id = id
+        self.name = name
+
+# Check if proxy env vars have been set
+try:
+    http_proxy = os.environ['http_proxy']
+except KeyError:
+    pp.pprint('No HTTP proxy found')
+    http_proxy = ''
+try:
+    trello_key = os.environ['TRELLO_KEY']
+    trello_token = os.environ['TRELLO_TOKEN']
+except KeyError:
+    pp.pprint('Trello env vars dont seem to be set')
+    trello_key = ''
+    trello_token = ''
+
+# Environment variables set for trell api
+try:
+    https_proxy = os.environ['https_proxy']
+except KeyError:
+    pp.pprint('No HTTPS proxy found')
+    https_proxy = ''
+
+proxyDict = { 
+            "http"  : http_proxy, 
+            "https" : https_proxy
+            }
+
+headers={}
+
+def get_item(id):
+    tempItem = requests.get(trello_endpoint+'cards/'+id+'?key='+trello_key+'&token='+trello_token, headers=headers, proxies=proxyDict).json()
+    tempItemListName = requests.get(trello_endpoint+'cards/'+id+'/list?key='+trello_key+'&token='+trello_token, headers=headers, proxies=proxyDict).json()
+    itemList = status(tempItem['idList'], tempItemListName['name'])
+    returnedItem = item(tempItem['id'],itemList, tempItem['name'])
+    return returnedItem
+
+def add_item(listId, title):
+    requests.post(trello_endpoint+'cards?name='+title+'&idList='+listId+'&key='+trello_key+'&token='+trello_token, headers=headers, proxies=proxyDict).json()
+
+def delete_item(id):
+    requests.delete(trello_endpoint+'cards/'+id+'?key='+trello_key+'&token='+trello_token, headers=headers, proxies=proxyDict).json()
+
+def save_item(id, idList, title):
+    requests.put(trello_endpoint+'cards/'+id+'?name='+title+'&key='+trello_key+'&idList='+idList+'&token='+trello_token, headers=headers, proxies=proxyDict).json()
+
+def get_app_board(app_board_name):
+    boards = requests.get(trello_endpoint+'members/'+trello_member+'/boards?key='+trello_key+'&token='+trello_token, headers=headers, proxies=proxyDict).json()
+    for x in range(len(boards)):
+        if boards[x]['name'] == app_board_name:
+            board = (boards[x]['name'], boards[x]['id'])
+    return board
```

Comment:
`board` might not be defined at this point.
The for loop is over complicated (even if you don't want a list comprehension)
All you wanted was the board id so you should just return the board id (and rename the function)
```
for board in boards:
    if board['name'] == app_board_name:
        return board['id']
raise a meaningful exception here
```